File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("https://training-support.net/")

    print("Title of Home page: ", driver.title)

    driver.find_element(By.ID, "about-link").click()

    print("Title of about us page: ", driver.title)

File: Selenium/Activities/Activity1.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();

        driver.get("https://www.training-support.net");
        System.out.println("Title of Home page: " + driver.getTitle());

        driver.findElement(By.id("about-link")).click();
        System.out.println("Title of about us page: " + driver.getTitle());

        driver.close();
    }
}
